app/mailers/communicart_mailer.rb

```diff
@@ -7,9 +7,11 @@ class CommunicartMailer < ActionMailer::Base
   add_template_helper ClientHelper
 
 
-  def proposal_notification_email(to_email, approval, show_approval_actions=true)
+  def proposal_notification_email(to_email, approval,
+                                  show_approval_actions=true, alert_partial=nil)
```

Comment:
I've split them into two methods (one calling the other) but kept a single template. If we find significant divergence, we can separate the templates further.